### DISCUSSION TITLE
cannot add multiple vlan in tagged field in interface or LAG

### DISCUSSION THREAD

**Discussion by jaheshkhan** [netbox-community/netbox#6137] [Q&A]
Im quite new to netbox. i was trying to create LAG. we have lot of LAGs in our switch. each lags have multiple allowed vlans. in netbox i was trying to add these tagged vlans in the LAG interface. i can only select each vlan one by one clicking one by one. as im trying to do existing network. its very time consuming process. we should be able to copy and paste multiple VLANs that are  already added in VLAN section. it should resolve the vlans automatically based on some kind of delimiters(";" ","  "space" "Enter" ) .

Or we can create vlan group and that group should be able to select . currently it again we need to individually select items in that group again. 

i dont know why its not included this way.

**Comment by dejantep** [comment]
Hello! 

When you edit LAG interface you have to change 802.1Q mode to "tagged" and then you'll have the ability to add multiple VLANs

**Reply by jaheshkhan** [reply]
That step i know. what i was looking for i want to do the same tasks on multiple interfaces. I really missed the point that i need to select multiple interfaces at a time and the click edit. then select mode mode tagged then enter multiple vlans. then click apply.
this will update more than one LAG interfaces at a time with same number of vlans. 

before finding this option i already spent more than 2 hours to do on each individual lags.

i was having about 35 LAGS with almost same VLANS
